documentation/src/main/asciidoc/ch08.asciidoc

```diff
@@ -10,8 +10,8 @@ approach via annotations. Let's have a look at these two document types.
 [NOTE]
 ====
 The XSD files are available via
-https://xmlns.jakarta.ee/xml/ns/validation/configuration and
-https://xmlns.jakarta.ee/xml/ns/validation/mapping.
+https://jakarta.ee/xml/ns/validation/configuration and
+https://jakarta.ee/xml/ns/validation/mapping.
```

Comment:
The schemas are online under the https://jakarta.ee/xml/ns/validation/ page. On that page is a table linking to the validation-mapping-3.0.xsd and validation-configuration-3.0.xsd files.

Reply:
Fixed.